mscore/workspace.cpp

```diff
@@ -40,6 +40,7 @@ QList<Workspace*> Workspace::_workspaces {};
 QList<QPair<QAction*, QString>> Workspace::actionToStringList {};
 QList<QPair<QMenu*  , QString>> Workspace::menuToStringList   {};
 
+const char* Workspace::fullWorkspaceTranslatableName{ QT_TRANSLATE_NOOP("Ms::Workspace", "Full") };
```

Comment:
@Jojo-Schmitz could you please remove the renaming from this PR? I'm ready to merge the changes related to Accordion palette and removing the Bagpipe embellishment, but renaming workspaces is something that needs additional discussion.

Reply:
There is no **renaming** workspaces, this 2nd commit is **adding** a 3rd workspace, generated on the fly.